assets/sass/registry.sass

```diff
@@ -5,6 +5,7 @@ $dark-blue: #39a9da;
 $core: #2C3ED3
 $instrumentation: #D32C3E
 $exporter: #3ED32C
+$tools: #2CD3B4
```

Comment:
I just generated this color with some tool online. Is there a better color available?

Reply:
it's probably fine for now